### ISSUE TITLE
Hide/Toggle Menu Bar Shortcut

### ISSUE THREAD

**Issue by incoheart** [drawpile/Drawpile#1387]
It would be cool if there were a shortcut to hide the top menu that contains File Edit View Selection Animation Tools Help. One could press **Ctrl + Shift + M** to toggle the menu bar. But this could be not assigned by default for accessibility reasons, so that the user knows what they are trying to remove from their user interface first.  It looks to me that the menu bar is there for accessibility reasons, since... it used to be possible to hide it in full screen, right?

This is what the artist does: First you have the scroll bars set to hidden under Preferences > User Interface > Miscellaneous, then you press F11 to fo full screen, then you press Tab to hide the docks, then you press Ctrl + Shift + M to disappear the top menu and now your canvas covers your whole screen.

I got the idea to toggle the menu from Qterminal. Qterminal does not have the ability to toggle the menu in its own top menu under View, so I don't think Drawpile should either, but it does have the shortcut assigned it in File > Preferences > Shortcuts . **Ctrl + Shift + M** is the default shortcut and it is already assigned, which I think Drawpile might not do if it is possible to implement this.
![qterminal](https://github.com/user-attachments/assets/5b269d50-b8e4-4e76-b3ec-047d53bc4570)
![qterminal_settings](https://github.com/user-attachments/assets/480fbf69-2a0e-4ed1-95bf-b5b0761221d9)

The Toggle Menu shortcut could be called **Hide Menu** instead of **Toggle Menu** to parallel it with **Hide Docks.**

**Comment by askmeaboutlo0m** [comment]
Yeah, it used to get removed in fullscreen, but not everyone has a keyboard or a mouse to whack the top edge of the screen with to bring the menu bar back, so it would leave you stuck in full screen forever with no way to get it back.

Adding a separate action that's only accessible via keyboard wouldn't have that issue and makes sense to have. Visual Studio Code and Firefox also let you hide the menu bar and summon it by pressing Alt once. Although a text editor can rely on you having a keyboard and Firefox also has a menu button always visible, so they can make this a persistent option instead of putting it behind a shortcut, which I don't think we can really do sensibly in Drawpile where you can hide every UI element except the canvas.

I'd probably not default-assign it to any shortcut though, since it seems like a pretty niche feature and accidentally activating it without knowing what you pressed would probably end up in having to restart the whole program to get out of it.
